### DISCUSSION TITLE
SkipTemplates in a packager versus SkipTemplates in a distribution [bug?]

### DISCUSSION THREAD

**Discussion by haydenbaker** [jreleaser/jreleaser#1583] [General]
I noticed that, when defining templates to skip in a packager at the top level, it doesn't actually do anything - the templates/files still get used and packaged:
```
    packagers {
        brew {
            active = 'ALWAYS'
            skipTemplates = ["README.*"]
        }
    }
```

However, when declaring template-skips under a distribution, they _do_ get skipped. 
```
    distributions {
        xyz {
            ...
            brew {
                active = 'ALWAYS'
                skipTemplates = ["README.*"]
            }
        ...
```

In the trace (for the first example), I see that the top-level packager has `skipTemplates` set, but in the packager for the distribution, `skipTemplates` is missing. FWIW, if I set a template-directory at the top-level, it gets propagated to the distribution-packager.

Is this a bug, or is this intended?

**Comment by aalmiray** [comment]
This looks like a bug indeed. All settings defined at the `packagers` section should be inherited by all distributions.

**Reply by aalmiray** [reply]
Filed https://github.com/jreleaser/jreleaser/issues/1584
